### PR TITLE
[2018-02] [sdk] sync ios and android changes from master into `2018-02`

### PR DESCRIPTION
**Pull request by lewurm**
we should try to be more careful about that in the future

### CODE REVIEW

**Comment by lewurm** [comment]
@monojenkins build OS X x64 Android SDK

**Comment by lewurm** [comment]
@monojenkins build OS X x64 iOS SDK

**Comment by lewurm** [comment]
@monojenkins build OS X x64 Android SDK

**Comment by lewurm** [comment]
@monojenkins build OS X x64 iOS SDK

File: scripts/ci/run-jenkins.sh [hunk 1]
```diff
@@ -94,6 +94,12 @@ if [[ ${CI_TAGS} == *'product-sdks-ios'* ]];
 	   exit 0
 fi
 
+if [[ ${CI_TAGS} == *'product-sdks-android'* ]];
+   then
+        ${TESTCMD} --label=runtimes --timeout=120m --fatal make -j4 -C sdks/builds package-android-{armeabi,armeabi-v7a,arme64-v8a,x86,x86_64} package-android-host-{Darwin,Linux,mxe-{Win32,Win64}} package-bcl
```
Comment (luhenry) [inline scripts/ci/run-jenkins.sh]: It's a mistake in master, please remove `package-host-Linux`.

**Comment by lewurm** [comment]
@monojenkins build OS X x64 Android SDK

**Comment by lewurm** [comment]
@monojenkins build OS X x64 iOS SDK

**Comment by lewurm** [comment]
why are the SDK lanes failing?

**Pull request merged**